webapp-django/crashstats/signature/static/signature/js/signature_tab_comments.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Tab for displaying comments table.
+ * Does not have any panels.
+ * Does not have any controls.
+ *
+ * @extends {SignatureReport.Tab}
+ * @inheritdoc
+ */
+SignatureReport.CommentsTab = function(tabName) {
+
+    var config  = {
+        'panels': false,
+        'dataDisplayType': 'table',
+        'pagination': true
+    };
+
+    SignatureReport.Tab.call(this, tabName, config);
+
+};
+
+SignatureReport.CommentsTab.prototype = SignatureReport.inherit(SignatureReport.Tab.prototype);
+
+// Extends getParamsForUrl to add the page parameter.
+SignatureReport.CommentsTab.prototype.getParamsForUrl = function () {
+
+    var params = SignatureReport.getParamsWithSignature();
+    params.page = this.page || SignatureReport.pageNum;
```

Comment:
If you use the config key ``pagination``, you can make that better and put it in the base class. Something like:

```javascript
var params = SignatureReport.getParamsWithSignature();
if (config.pagination) {
  params.page = this.page || SignatureReport.pageNum;
}
return params;
```